packages/zoltan2/core/src/input/Zoltan2_BasicIdentifierAdapter.hpp

```diff
@@ -50,16 +50,19 @@
 #ifndef _ZOLTAN2_BASICIDENTIFIERADAPTER_HPP_
 #define _ZOLTAN2_BASICIDENTIFIERADAPTER_HPP_
 
+#include <Kokkos_Core.hpp>
 #include <Zoltan2_IdentifierAdapter.hpp>
 #include <Zoltan2_StridedData.hpp>
 
+using Kokkos::ALL;
```

Comment:
We probably shouldn't call `using` in a header file

Reply:
Done